csrc/ops/arith.cpp

```diff
@@ -89,9 +89,6 @@ TensorView* bitCastOp(DataType dtype, TensorView* v1) {
 }
 
 Val* unaryOp(UnaryOpType type, Val* v1) {
-  TORCH_INTERNAL_ASSERT(
-      type != UnaryOpType::Address,
-      "The reference operator & is not accessible in the Fusion IR");
```

Comment:
Nothing is failing, but I just don't like this check. I think it is totally fine to have it in fusion IR, especially for testing or debugging purposes.